src/test/kotlin/graphics/scenery/tests/examples/volumes/OutOfCoreRAIExample.kt

```diff
@@ -43,7 +43,7 @@ class OutOfCoreRAIExample: SceneryBase("Out-of-core RAI Rendering example", 1280
         scene.addChild(cam)
 
         // download example TIFF stack, T1 head example from NIH
-        val imp: ImagePlus = IJ.openImage("https://imagej.nih.gov/ij/images/t1-head.zip")
+        val imp: ImagePlus = IJ.openImage(getDemoFilesPath() + "/volumes/t1-head.zip")
```

Comment:
This .zip is wrong

Reply:
You might have an old models.zip file. The current one has both the .zip and the .xml/.h5 👍